SparkPost/2web2ui
# TR-2891: Update RV api integration page to include delivery_confidence score

Refer to [TR-2891](https://sparkpost.atlassian.net/browse/TR-2891)

### What Changed
 - Added "delivery_confidence" to API Integration tab static example

<img width="1386" alt="Screen Shot 2020-11-20 at 4 47 21 PM" src="https://user-images.githubusercontent.com/1335605/99853034-374b6c80-2b50-11eb-9e07-2fb9b06bd6c8.png">

### How To Test
 - Open http://app.sparkpost.test/recipient-validation/api
 - Confirm "delivery_confidence" is present in the example response
